themes/default/content/blog/managing-github-with-pulumi/index.md

```diff
@@ -0,0 +1,599 @@
+---
+title: "How we manage GitHub at Pulumi with Pulumi"
+date: 2022-03-18T15:29:33-07:00
+meta_desc: In this article, Guinevere Saenger describes how Pulumi migrated management of GitHub team permissions using Pulumi all the way down.
+authors: ["guinevere-saenger"]
+tags: ["go", "migration", "packages"]
```

Comment:
What do you think about a `github-actions` tag? I think it's reasonable to add that one. And possibly a new tag for `ci-cd` or something like that. Thoughts?

Reply:
I think a `github-actions` tag would be really dope, a `ci-cd` would be cool overall but not really applicable here imo - we're just _using_ ci/cd, not really showing an example of building new ci/cd?

Reply:
I think I'd still add it as someone can use the tag to see what's possible. There's already a `github-actions` tag, so go ahead and add it! And feel free to add a `ci-cd` one. The tag list in the blogging guide is just a rough outline. Tags won't show up in the tag cloud until they've got a handful of articles on them.

Reply:
would a people-ops be a useful tag?